src/components/ArticlePage.jsx

```diff
@@ -0,0 +1,26 @@
+import getAnArticle from "../utils/getAnArticle";
+import { useState ,useEffect } from "react";
+import Loading from "./Loading";
+import { useParams } from 'react-router-dom'
+import formatDate from "../utils/FormatDates";
+import Comments from "./Comments";
+
+const ArticlePage = () => {
+    const [article,setArticle] = useState({});
```

Comment:
we should have a loading state to provide visual feedback to the user on slow load